ui/app/components/dashboard/quick-actions-card.js

```diff
@@ -74,15 +83,20 @@ export default class DashboardQuickActionsCard extends Component {
           buttonText: 'View certificate',
           model: 'pki/certificate/base',
           route: 'vault.cluster.secrets.backend.pki.certificates.certificate.details',
+          nameKey: 'id',
```

Comment:
this should be `commonName` https://github.com/hashicorp/vault/blob/0b12cdcfd17278d7cca25ec37ffacd96b8d5c6c7/ui/app/models/pki/certificate/base.js#L41

Reply:
Since commonName isn't required for certs, it defaults to the cert ids but it's greyed out here I think because it's expected a nameKey?
<img width="546" alt="Screenshot 2023-08-30 at 1 14 36 PM" src="https://github.com/hashicorp/vault/assets/30884335/78741248-9e04-40ed-b05a-4aaa5b3568f4">

Reply:
Oh yeah for certificates we don't have any data keys so common name isn't available in any LIST requests. I forgot! `nameKey` can just be removed all together then

Reply:
Makes sense - it also happens for issuerName too if an issuerName hasn't been set in pki 😭